### ISSUE TITLE
How to identify the offiline operators if 1/2 operators are malicious in a cluster

### ISSUE THREAD

**Issue by TinyVege** [bloxapp/ssv-spec#351]
for example, Distributed Validator A is  handled by operator1,2,3,4. 
We assume operators1,2 are honest while operators 3-4  is malicious and pretend to be offline, so how can the ssv identify exactly the malicous operators 3-4.

**Comment by GalRogozinski** [comment]
See [BFT Forensics](https://decentralizedthoughts.github.io/2020-11-19-bft-protocol-forensics/)

**Comment by TinyVege** [comment]
> See [BFT Forensics](https://decentralizedthoughts.github.io/2020-11-19-bft-protocol-forensics/)

I have read it. It only identifies the double vote misbehavior, can not solve the liveness vioaltion right?

My question is that operators3-4 don't send any protocol(validator duty) messages to the dvt cluster and discard the protocol messages of operators1-2. How to prove the operators3-4 are the faultys ?

**Comment by MatheusFranco99** [comment]
Yes, BFT Forensics can't solve the protocol. On a theoretical point of view, any BFT can only guarantee its properties (agreement, termination, etc) if at maximum $f$ nodes are malicious.

We don't have BFT Forensics yet, but multiple strategies could be added, such as idleness, invalid messages and so on.

**Comment by TinyVege** [comment]
I'm also interested in evaluating the performance of operators in ssv (shown bolow)
> https://docs.ssv.network/learn/performance
![image](https://github.com/bloxapp/ssv-spec/assets/106942308/16848bd5-4ad3-4ab9-828d-64f85392e327)
https://docs.ssv.network/learn/performance/statuses
![image](https://github.com/bloxapp/ssv-spec/assets/106942308/8a1d85f7-979d-4ac7-bfce-2725804f94db)


Now assume, there are four operator1, operator2,  operator3,  *operator4. '*' implies the Byzantine. 
And operaotors1-4 only perform the validatorX.
Operators1-3 do the protocol  while the operator4 is offline --> The validatorX does not miss any duty.

⚠⚠⚠
**_So the result is that the performance and status of operator1-4 are the same : 100% and active_ 
Right ?** 👀👀👀
@MatheusFranco99 @GalRogozinski

**Comment by GalRogozinski** [comment]
Your last question is best to be asked on our Discord server
https://discord.com/invite/ssvnetworkofficial

this repo is used for discussing issues of the core protocol itself

**Comment by TinyVege** [comment]
ok ,thanks
